src/components/home/partners/Partners.jsx

```diff
@@ -8,10 +8,10 @@ const Partners = () => {
     <section>
       <Typography
         component={'h2'}
-        variant={'h3'}
-        padding={{ xs: '50px 0 100px 0', md: '50px 50px 150px 50px' }}
+        padding={{ xs: '25px 0 100px 0', md: '50px 50px 100px 50px' }}
         display={'flex'}
         textAlign={'center'}
+        sx={{ typography: { xs: 'h4', sm: 'h3' } }}
```

Comment:
Is this instead of creating a ternary for the `variant`? Pretty neat.

Not sure why `variant={{xs: 'h4', sm: 'h3'}}` doesn't produce same result, but it doesn't.

Reply:
Yeah, I found it after searching around.

And yeah seems like that should work but I can't get it to.